### PR TITLE
Add styling changes to everything except Modal

### PR DESCRIPTION
**Pull request by tfettig22**
Whats this PR do?
- Add styling updates to index, App, Header, MovieContainer, and Card
- Re-format Cards so that the title rating and logo is displayed under the poster image
- Can't figure out why MovieContainer isn't filling the entire page, currently has a red border we can see about half way down the movie cards.  This means the cards are overflowing and could cause issues later on.

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -25,12 +25,10 @@ class App extends Component {
       
   render = () => {
     return (
-      <main className = 'App'>
+      <main className = 'app'>
         <Header />
-        <Switch>
-          <Route exact path="/" render={ () => <MovieContainer movies={this.state.movies}/> } />
```
Comment (GOECHA) [inline src/components/App/App.js:31]: This refactor looks great.

File: public/index.html [hunk 2]
```diff
@@ -28,7 +28,7 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
+    <div id="root" class="root"></div>
```
Comment (Maia-Cochran) [inline public/index.html:31]: great catch on the 'root' :-)

File: src/components/Card/Card.css [hunk 3]
```diff
@@ -1,11 +1,10 @@
 .movie-card {
   display: flex;
-  justify-content: flex-end;
-  height: 20%;
```
Comment (GOECHA) [inline src/components/Card/Card.css:4]: > 🟢 Nice refactor!

File: src/components/Card/Card.js [hunk 4]
```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 import './Card.css';
 
-const Card = ({ logo, posterImage, rating, id, selectAMovie }) => {
```
Comment (GOECHA) [inline src/components/Card/Card.js:4]: Good catch, from previous refactor.

**Review by Maia-Cochran — APPROVED** [review]

File: src/components/Modal/Modal.css [hunk 5]
```diff
@@ -108,7 +102,7 @@ p {
     font-size: small;
 }
 
-.rating{
+.modal-rating{
```
Comment (GOECHA) [inline src/components/Modal/Modal.css:105]: thank you for updating this for dev empathy

File: src/components/Modal/Modal.js [hunk 6]
```diff
@@ -20,7 +20,7 @@ class SingleMovie extends Component {
 
   render = () => {
     let movie = this.state.movie[0]
-    if (this.state.movie[0] === undefined) {
+    if (movie === undefined) {
```
Comment (GOECHA) [inline src/components/Modal/Modal.js:23]: This refactor looks great.

**Review by GOECHA — APPROVED** [review]
Great job with functionality, this looks clean and is very readable. 
> 🟢 Nice refactor!

**Pull request merged**